01/app.js

```diff
@@ -0,0 +1,8 @@
+const user = {
+    firstName: 'Anna',
+    lastName: 'Kołodziej',
+    sex: 'female',
+    age: 32,
+}
+
+console.log(user);
```

Comment:
bardziej chodziło, o to, aby wyświetlić każdy z osobna czyli `console.log(user.firstName)`